README.md

```diff
@@ -78,64 +81,169 @@ You can run the examples by cloning the repo or copy pasting the below sample in
 ## Example 1 - CBS Authentication
 
```

Comment:
You may want to somehow indicate (maybe in the Example 1 header) that this code block is referred to later as `./cbsAuth`.